themes/digital.gov/layouts/partials/core/footer.html

```diff
@@ -16,7 +16,7 @@
 <footer class="usa-footer usa-footer--big" role="contentinfo">
   <div class="usa-footer__primary-section">
     <div class="grid-container grid-container-desktop">
-      <nav class="usa-footer__nav">
+      <nav class="usa-footer__nav" aria-label="Footer">
```

Comment:
Let's improve context by adding the word navigation to the aria label.

```suggestion
      <nav class="usa-footer__nav" aria-label="Footer navigation">
```

Reply:
I've updated the code to your suggestion